sensor_msgs/Image in_img
---
sensor_msgs/Image out_img
float32[] x
float32[] y
float32[] z